src/time-map.scm

```diff
@@ -103,11 +127,27 @@
   (cog-filter 'NumberNode (show-visible-faces))
 	;;(string-concatenate (map (lambda (x)(string-append (cog-name x) " ")) (cog-filter 'NumberNode (show-visible-faces)) ))
 )
+;(define (get-visible-faces)(list (NumberNode "1")(NumberNode "2")))
 ;;below returns face id of face nearest to sound vector atleast 10 degrees, or 0 face id
 (define (snd1-nearest-face)
-	(let* ((lst (get-visible-faces))(falist (map (lambda (x)(list (string->number x) (angle_face_snd1 x))) lst)))
-		(if (< (length falist) 1) 0 (let* ((alist (map (lambda (x)(cdr x))falist))(amin (abs (min alist))))
-			(if (< (/ (* 3.142 10.0) 180.0) amin) (car (car(filter (lambda (x)(< amin (+ (abs (car x)) 0.0001)))alist))) 0))
+	(let* ((lst (get-visible-faces))
+				 (falist (map (lambda (x)(list (string->number (cog-name x)) (angle_face_snd1 (cog-name x)))) lst)))
```

Comment:
bad indentation. Also, please wrap lines so that they fit into 80 columns -- not everyone has wide terminals